NEWS.txt

```diff
@@ -61,6 +61,14 @@ Upgrading
       to be done anything upon upgrade. Furthermore, IMDS of version 2 can be configured to be required in AWS EC2 console.
       Consult cassandra-rackdc.properties for more details. (CASSANDRA-16555)
 
+Deprecation
+---------
+    - Chronicle Queue has changed the enums used for log rolling (cassandra.yaml -> full_query_logging_options:roll_cycle).
+    Older legacy options will still work for the foreseeable future but you will see warnings in logs and future dependency
+    upgrades may break your log rolling param. The default log rolling param will be changed with the next major release
+    from HOURLY to FAST_HOURLY, primarily different on how frequently indexes are built (256 in FAST_HOURLY vs. 16 in HOURLY).
+    For more info refer to: net.openhft.chronicle.queue.RollCycles
```

Comment:
how come we didn't add this, and the `NOTE:`  in cassandra.yaml, upstream ?  
was it just an oversight ?

Reply:
Different versions, no oversight there (also it would have popped up in CI for which I am going to prepare a patch now in cassandra-dtest)
https://github.com/apache/cassandra/blob/cassandra-5.0/.build/parent-pom-template.xml#L841-L924
https://github.com/datastax/cassandra/blob/main/build.xml#L159-L164

This change was added in 5.24.x in this commit - https://github.com/OpenHFT/Chronicle-Queue/commit/05faa68f63465e98c7f255ac2ce8e260d187d6a6

I skimmed some of the commits here - https://github.com/OpenHFT/Chronicle-Queue/compare/chronicle-queue-5.23.37...chronicle-queue-5.24ea27, but I am afraid for full scan of all changes for all dependencies we need to open follow up ticket. I am fixing the immediate problems here, and I won't have the time for more than that. I will ping some of the other teams internally.